.github/workflows/pre-release.yml

```diff
@@ -20,22 +20,16 @@ jobs:
         with:
           go-version: '1.18'
 
-      - name: Build
-        uses: goreleaser/goreleaser-action@v4
-        if: ${{ github.event_name == 'pull_request' }}
-        with:
-          version: latest
-          args: build --skip-validate   # skip validate skips initial sanity checks in order to be able to fully run
-
-      # Link to CHANGELOG_PENDING.md as release notes.
-      - run: echo https://github.com/cometbft/cometbft/blob/${GITHUB_REF#refs/tags/}/CHANGELOG_PENDING.md > ../release_notes.md
+      - run: |
+          VERSION="${GITHUB_REF#refs/tags/}"
+          CHANGELOG_URL="https://github.com/cometbft/cometbft/blob/${VERSION}/CHANGELOG.md"
+          echo "See the [CHANGELOG](${CHANGELOG_URL}) for changes available in this pre-release, but not yet officially released." > ../release_notes.md
```

Comment:
This attempts to clarify what we're doing here by building the `release_notes.md` file.